### PR TITLE
recreate dockerfile branch with signed commits

### PR DESCRIPTION
**Pull request by anwilkie**
# What and why?
This PR moves the dockerfile to the _infra/docker directory and adds signed commits.

In addition, the certifi dependency has been updated.

# How to test?
Check GA build.

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-313)

### CODE REVIEW

**Review by arroyoAle — APPROVED** [review]
looks good, build works, and docker compose works as expected

File: Pipfile.lock
Comment (matthew-robinson-ons) [inline Pipfile.lock:1]: Was this supposed to be updated?

**Review by matthew-robinson-ons — CHANGES_REQUESTED** [review]
There are a lot of changes in the pipfile.lock which i can't see documented. Will be hard to work out what has gone wrong if something breaks.

File: Pipfile.lock
Comment (anwilkie) [reply Pipfile.lock:1]: Yes, there were vulnerabilities that required dependency updates.

Comment (anwilkie) [reply Pipfile.lock:1]: I ran commands to update the specific packages in question. I don't know why the other packages got updated; I think that's just how pipenv works sometimes. If you're concerned about those updates, feel free to deploy the PR to your environment and test it out.

Comment (matthew-robinson-ons) [reply Pipfile.lock:1]: You can use `pipenv upgrade` to update a specific package and use pipenv graph to display the dependencies so you can target the correct package and subpackages. Your card doesn't have any runtime changes but by updating all those packages it now does and the testing for it is a lot more in-depth than 'check GA build'. Aiming to make the smallest number of necessary changes seems t]like the better approach to me than just checking in a load of stuff that we don't know the effects of.

**Review by matthew-robinson-ons — APPROVED** [review]
Looks good and dependencies/vulnerabilites updated

**Pull request merged**